#两轮距离317  0.605
wheel_base: 0.2796
##车轮半径:  直径：0.375 0.16975
wheel_radius: 0.105
##齿轮的齿数
num: 80

# time =  1630396208.573587418
# robot position = 2.18272, -0.54178, 
# robot Matrix =  0.46692  0.88422 -0.01202  2.18272
# -0.88429  0.46684 -0.00926 -0.54178
# -0.00258  0.01495  0.99988  0.25108
#  0.00000  0.00000  0.00000  1.00000
# yaw -3.12664